lib/Cake/Console/ShellDispatcher.php

```diff
@@ -177,6 +177,9 @@ public function setErrorHandlers() {
 		}
 		set_exception_handler($exception['consoleHandler']);
 		set_error_handler($error['consoleHandler'], Configure::read('Error.level'));
+
+		App::uses('Debugger', 'Utility');
```

Comment:
This don't work. Class App isn't declared until ShellDispatcher::_bootstrap be called :(. I will revert my commit.